common/djangoapps/django_comment_common/utils.py

```diff
@@ -143,7 +143,12 @@ def set_course_discussion_settings(course_key, **kwargs):
     Returns:
         A CourseDiscussionSettings object.
     """
-    fields = {'division_scheme': basestring, 'always_divide_inline_discussions': bool, 'divided_discussions': list}
+    fields = {
+        'division_scheme': basestring,
+        'always_divide_inline_discussions': bool,
+        'divided_discussions': list,
+        'discussions_id_map': dict,
```

Comment:
Will this validation blow up right after the migration is run and the value for the `discussions_id_map` is still null?

Reply:
No, `set_course_discussion_settings` will never get called with a `discussions_id_map` kwarg that is set to None.